nb-extension/js/init/init.js

```diff
@@ -16,7 +16,9 @@ define([
 
     // Enable shadow dom if it is there for polymer elements.
     window.Polymer = window.Polymer || {};
-    window.Polymer.dom = 'shadow';
+    //  NOTE: This breaks some code because it is expecting Polymer.dom to be a function
+    // window.Polymer.dom = 'shadow';
+    window.Polymer.dom = window.Polymer.dom ? window.Polymer.dom : 'shadow';
```

Comment:
@deedubbu There many be a different way of setting `shadow` dom since now Polymer moved the `dom` api to the top of the `Polymer` global.